rules/S6875/java/metadata.json

```
@@ -0,0 +1,24 @@
+{
+  "title": "SequencedCollection method should be used to add or remove first or last element",
+  "type": "CODE_SMELL",
+  "status": "ready",
+  "remediation": {
+    "func": "Constant\/Issue",
+    "constantCost": "5min"
+  },
+  "tags": [
+    "Java 21"
+  ],
+  "defaultSeverity": "Major",
+  "ruleSpecification": "RSPEC-6875",
+  "sqKey": "S6875",
+  "scope": "All",
+  "defaultQualityProfiles": ["Sonar way"],
+  "quickfix": "unknown",
+  "code": {
+    "impacts": {
+      "MAINTAINABILITY": "HIGH"
```

I would propose only `MEDIUM` here:
```suggestion
      "MAINTAINABILITY": "MEDIUM"
```